.github/workflows/ci.yml

```
@@ -213,117 +213,122 @@ jobs:
+          fi
+        env:
+          GH_TOKEN: ${{ github.token }}
 
-          cp -r "$TMP"/* .
-          git add -A
-          git commit -m "Screenshot diffs for PR #${{ github.event.pull_request.number }}"
-          COMMIT_SHA=$(git rev-parse HEAD)
-          git push origin "$BRANCH" --force
+      - name: Close stale update PR (no diffs)
+        if: steps.diff.outputs.count == '0' && steps.check.outputs.should_run == 'true' && steps.screenshots.outcome == 'success'
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)